ec2/sc-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -18,18 +18,21 @@ Metadata:
         default: Disk Size
 
 Mappings:
-
   NotebookTypes:
     Rstudio:
       AMIID: "ami-051310d409a32a5aa"  # https://github.com/Sage-Bionetworks-IT/packer-rstudio/tree/v2.0.0
+  AccountToImportParams:
```

Comment:
It would be even better to move the map to another template and export it.. then import the key/value every time we need to use it.

Reply:
> It would be even better to move the map to another template and export it.. then import the key/value every time we need to use it.

hmm, interesting idea @ahayden.  i'm not sure what that would look like.  Maybe you can do a PR for it so we can discuss your idea in the PR?

Reply:
I like that idea, and an AWS::Include can be used in the mappings section of the template (just not in the parameters section)

Reply:
still not seeing how `AWS::Include` would help make this better.  looks like you guys know something i cannot see.  I'm looking forward to a PR for enlightenment.